umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -92,41 +100,34 @@ For any other microservice, follow these steps to enable the LoadBalancer servic
    ```
 
    Replace `<external-ip>` with the external IP address of the node.
-4. Scroll to the `spec.type` section and change the value from ClusterIP to
-   LoadBalancer.
-5. Click **Save** to apply the changes.
+4. Go to the `spec.type` section and change the value from `ClusterIP` to
+   `LoadBalancer`. 
+5. After saving, your changes will be applied automatically and the service will 
+   be updated. Now, you can access the service with the configured address.
 
-If you installed the United Manufacturing Hub on your local machine, either
-using the Management Console or the command line, you also need to map the port
-exposed by the k3d cluster to a port on your local machine. To do that, run the
-following command:
+### Port forwarding
 
-```bash
-k3d cluster edit {{< resource type="cluster" name="name" >}} --port-add "<local-port>:<cluster-port>@server:0"
-```
-
-Replace `<local-port>` with a free port number on your local machine, and
-`<cluster-port>` with the port number of the service.
+If you don't want to create a LoadBalancer service, effectively exposing the
+microservice to anyone that has access to the host IP address, you can forward 
+the port to your local machine.
 
-### Port forwarding in {{% resource type="lens" name="name" %}}
+1. Execute the following command to get the list of available services and remember 
+   the service to be exposed.
+   ```bash
+   sudo $(which kubectl) get svc -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
+   ```
+2. Run the following command to forward the port:
+   ```bash
+   sudo $(which kubectl) port-forward service/<your-service> <local-port>:<remote-port> -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
+   ```
```

Comment:
Explain what are the ports:

"Where `<local-port>` is the port on the host that you want to use, and `<remote-port>` is the service port that you noted before. Usually, it's good practice to pick a high number (greater than 30000) for the host port, in order to avoid conflicts."

Reply:
updated